app/controllers/user/dashboard_controller.rb

```diff
@@ -1,13 +1,8 @@
 class User::DashboardController < ApplicationController
-  before_action :require_current_user
+  before_action :require_current_user,
+                :exclude_admin
+
   def show
-    NpcModel.where(is_saved: 0).ids.map do |id|
-      NpcModelArmor.where(npc_model_id: id).destroy_all
-      NpcModelPack.where(npc_model_id: id).destroy_all
-      NpcModelWeapon.where(npc_model_id: id).destroy_all
-      NpcModelSpell.where(npc_model_id: id).destroy_all
-    end
-    NpcModel.where(is_saved: 0).destroy_all
```

Comment:
I took this out so that, in the future, someone can save an NPC even if they're not logged in yet.